lib/active_merchant/billing/gateways/flow.rb

```diff
@@ -174,6 +176,23 @@ def store input
         response.token
       end
 
+      # Creates and order
+      # https://docs.flow.io/module/localization/resource/orders#post-organization-orders
+      def flow_create_order body, query_string={}
+        flow_instance.orders.post @flow_organization, body, query_string
+      rescue => exception
+        error_response exception
+      end
+
+      # Submits an order and
+      # pushes all subsequent authorizations from status "review" to "authorized"
```

Comment:
this comment is not necessarily true - when the order is submitted, it will be queued from Fraud check, but the outcome is not always `authorized`